### ISSUE TITLE
Display the identity records information in the dashboard

### ISSUE THREAD

**Issue by codeDavidLopez** [amosproj/amos2022ss06-idunion-blockchain-dashboard#45]
## User story
1. As a user
2. I want / need the data of the identity records displayed
3. So that a overview of the identity records is provided

## Acceptance criteria
* The ledger entity data is defined and processed by the metrics engine
* The data of the identity records is stored in Prometeus.io and available
* The Grafana dashboard is extended with the identify record information

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## Feature DoD 
* Feature has been fully implemented
* All acceptance criteria were met
* All tests are passing
* No known errors
* Documentation updated
* Bill of materials updated

## Sprint Release DoD
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Peer code review passed
* Developers agreed to release
* Documentation updated
* Bill of materials updated
